examples/doctest/CMakeLists.txt

```diff
@@ -4,7 +4,7 @@ include("../common.cmake")
 
 project(download-doctest)
 
-set(CMAKE_CXX_STANDARD 98)
+set(CMAKE_CXX_STANDARD 11)
```

Comment:
Actually this line should be removed completely, standard should be set in toolchain, see:
* https://docs.hunter.sh/en/latest/overview/customization/toolchain-id.html